lib/rules/sotd/obsl-rescind.js

```diff
@@ -0,0 +1,53 @@
+// SotD
+// <p>W3C has chosen to (rescind|obsolete) the Sample Specification Recommendation for the
+// following reasons: [...list of reasons...]. For additional information about
+// replacement or alternative technologies, please refer to the <a
+// href="https://www.w3.org/2016/11/obsoleting-rescinding/">explanation of
+// Obsoleting and Rescinding W3C Specifications</a>.</p>
```

Comment:
Ah sorry, I forgot to mention that. That text was suggested by @koalie while I was preparing the mockups for [obsolete recs](https://www.w3.org/2017/02/tr-rscnd-obsl/obsoleted.html) and [rescinded recs](https://www.w3.org/2017/02/tr-rscnd-obsl/rescinded.html).